Uni_life_tracker/app/src/main/res/layout/main_nav_drawer.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="120dp"
+            android:layout_margin="15dp"
+            android:onClick="ClickLogo"
+            android:src="@drawable/ic_menu"
+            android:contentDescription="@string/app_name"/>
+
+    <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/app_name"
+            android:textSize="20sp"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textStyle="bold"
+            android:textAlignment="center"
+            android:textColor="@color/black"
+            android:layout_marginBottom="20dp"
+    />
+
+<!-- line divider -->
+<!--    <View-->
+<!--            android:layout_width="match_parent"-->
+<!--            android:layout_height="1dp"-->
+<!--            android:layout_marginTop="16dp"-->
+<!--            android:layout_marginBottom="16dp"-->
+<!--            android:background="@color/lavender"-->
+<!--    />-->
+
+    <LinearLayout android:layout_width="match_parent"
```

Comment:
You have a couple of styling errors in this file that say "This tag and its children can be replaced by one `<TextView/>` and a compound drawable"